core/bin/dust_api.rs

```diff
@@ -136,6 +137,21 @@ impl APIState {
                 let manager = self.run_manager.lock();
                 utils::info(&format!("{} pending runs", manager.pending_runs.len()));
             }
+            // Roughly every 4 minutes, cleanup dead SQLite workers if any.
+            if loop_count % 65536 == 0 {
```

Comment:
Why that much?

Reply:
Didn't feel super useful to do it frequently (it should be an edge case, because it's only for crashed workers). Do you think it should be more frequent ?

Reply:
It's almost free to run so I would do it more frequently but that makes no big difference assuming they get pruned when something tries to schedule something on a dead one.